FFmpegInterop/Utility.h

```diff
@@ -56,7 +56,7 @@ namespace winrt::FFmpegInterop::implementation
 {
 	inline std::string tolower(_Inout_ std::string str)
 	{
-		std::transform(str.begin(), str.end(), str.begin(), [](_In_ char c){ return std::tolower(c); });
+		std::transform(str.begin(), str.end(), str.begin(), [](_In_ char c){ return static_cast<char>(std::tolower(c)); });

```

Comment:
nit: looks right to me, but looks different than the sample implementation at https://en.cppreference.com/w/cpp/string/byte/tolower